src/components/ContentHighlights/data/constants.js

```diff
@@ -1,47 +1,84 @@
-/* eslint-disable*/
-
-export const TEST_COURSE_HIGHLIHTS_DATA = [
+// Stepper Step Text that match testing components
+export const STEPPER_STEP_TEXT = {
+  createTitle: 'Create a title for the highlight collection',
+  selectCourses: 'Select Courses to Add',
+  confirmContent: 'Confirm your Content',
+  confirmHighlight: 'Confirm your Highlight',
+};
+// Default footer values based on API response for ContentHighlightCardItem
+export const FOOTER_TEXT_BY_CONTENT_TYPE = {
+  course: 'Course',
+  program: 'Program',
+  learnerpathway: 'Pathway',
+};
+// badge status for published vs unpublished highlight cards
+export function HIGHLIGHT_CARD_BADGE_STATUS(isPublished) {
```

Comment:
nit (though you should defer on it since the badges will be removed anyways in favor of the distinct "Published" vs. "Drafts" sections):

```
export const HIGHLIGHT_CARD_BADGE_STATUS = {
  PUBLISHED: { label: 'Published', variant: 'primary' },
  DRAFT: { label: 'Draft', variant: 'light' },
}
}
```